hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2866,6 +2866,15 @@
     </description>
   </property>
 
+  <property>
+    <name>ozone.om.lock.metrics.collection.enable</name>
+    <tag>OZONE, OM</tag>
+    <value>false</value>
+    <description>Defaults to false. If true, the OMLockMetrics collection is
+      enabled. If false, it is disabled.
```

Comment:
```suggestion
    <description>If set to true enables OMLockMetrics collection. Defaults to false/disabled.
```

Reply:
Thank you for the review, I have made the changes!